doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
Hmm... sounds like we need a sphinx expert to chime in here with their advice.  I'm not heavily opinionated about how this goes, I just like the idea of keeping the manual pages for the plugins.  I can imagine that that part might be of particular interest to end-users browsing the manual.

I guess what I am wondering is whether we can simply modify our sphinx build to extract and create the RST subpages prior to running sphinx, that way we we keep the plugin pages *and* have the documentation only in the plugin strings.  If the nesting is too deep, we could just create the pages at a lower level in the hierarchy.